package.json

```diff
@@ -42,69 +41,48 @@
   ],
   "devDependencies": {
     "@types/algoliasearch": "3.34.11",
-    "algolia-frontend-components": "0.0.35",
     "algoliasearch": "4.8.3",
-    "babel-core": "6.26.3",
-    "babel-loader": "6.4.1",
-    "babel-preset-es2015": "6.24.1",
-    "browserify": "14.5.0",
-    "bulkify": "1.4.2",
-    "collect-json": "1.0.8",
-    "colors": "1.1.2",
-    "conventional-changelog-cli": "1.3.2",
     "doctoc": "1.3.0",
+    "patch-package": "6.2.2",
     "eslint": "1.10.3",
     "eslint-config-airbnb": "0.0.8",
     "eslint-config-algolia": "3.0.0",
-    "exorcist": "1.0.1",
+    "onchange": "3.2.1",
     "gh-pages": "1.0.0",
-    "gulp": "3.9.1",
-    "gulp-cli": "1.3.0",
-    "gulp-livereload": "3.8.1",
-    "gulp-sass": "2.3.2",
-    "handlebars": "4.1.0",
     "jest": "24.7.1",
     "jest-watch-typeahead": "0.3.0",
+    "browserify": "14.5.0",
+    "watchify": "3.7.0",
+    "exorcist": "1.0.1",
+    "uglify-js": "2.8.29",
+    "colors": "1.1.2",
+    "conventional-changelog-cli": "1.3.2",
+    "pretty-bytes-cli": "2.0.0",
+    "prompt": "1.0.0",
+    "semver": "5.3.0",
+    "shelljs": "0.7.8",
+    "mversion": "1.13.0",
+    "yargs": "13.2.4",
+    "collect-json": "1.0.8",
     "jsdoc-parse": "1.2.7",
-    "jsdoc-to-markdown": "1.3.9",
+    "requizzle": "0.2.3",
```

Comment:
https://github.com/hegemonic/requizzle/issues/6, likely fixed by update to jsdoc-parse too, to check